src/db/migration/V1_1727687613772__Data_export_views.sql

```diff
@@ -0,0 +1,468 @@
+       oh.tep_kasitelty                        AS tep_kasitelty,
+       oh.osa_aikaisuustieto                   AS osa_aikaisuustieto,
+       oh.oppisopimuksen_perusta_koodi_uri     AS oppisopimuksen_perusta_koodi_uri,
+       oh.oppisopimuksen_perusta_koodi_versio  AS oppisopimuksen_perusta_koodi_versio,
+       oh.yksiloiva_tunniste                   AS yksiloiva_tunniste,
+       oh.tyopaikalla_jarjestettava_koulutus_id AS tyopaikalla_jarjestettava_koulutus_id
+FROM osaamisen_hankkimistavat oh
+         JOIN yhteisen_tutkinnon_osan_osa_alueen_osaamisen_hankkimistavat ytooaoh
+              on oh.id = ytooaoh.osaamisen_hankkimistapa_id
+         JOIN yhteisen_tutkinnon_osan_osa_alueet ytooa
+              on ytooaoh.yhteisen_tutkinnon_osan_osa_alue_id = ytooa.id
+         JOIN hankittavat_yhteiset_tutkinnon_osat yto
+              on ytooa.yhteinen_tutkinnon_osa_id = yto.id
+UNION
+SELECT oh.id                                   AS id,
+       oh.created_at                           AS created_at,
+       oh.updated_at                           AS updated_at,
+       oh.deleted_at                           AS deleted_at,
+       ato.hoks_id                             AS hoks_id,
+       concat('ato_', ato.id)                  AS osa_id,
+       oh.jarjestajan_edustaja_nimi            AS jarjestajan_edustaja_nimi,
+       oh.jarjestajan_edustaja_rooli           AS jarjestajan_edustaja_rooli,
+       oh.jarjestajan_edustaja_oppilaitos_oid  AS jarjestajan_edustaja_oppilaitos_oid,
+       oh.ajanjakson_tarkenne                  AS ajanjakson_tarkenne,
+       oh.osaamisen_hankkimistapa_koodi_uri    AS osaamisen_hankkimistapa_koodi_uri,
+       oh.osaamisen_hankkimistapa_koodi_versio AS osaamisen_hankkimistapa_koodi_versio,
+       oh.hankkijan_edustaja_nimi              AS hankkijan_edustaja_nimi,
+       oh.hankkijan_edustaja_rooli             AS hankkijan_edustaja_rooli,
+       oh.hankkijan_edustaja_oppilaitos_oid    AS hankkijan_edustaja_oppilaitos_oid,
+       oh.alku                                 AS alku,
+       oh.loppu                                AS loppu,
+       oh.module_id                            AS module_id,
+       oh.tep_kasitelty                        AS tep_kasitelty,
+       oh.osa_aikaisuustieto                   AS osa_aikaisuustieto,
+       oh.oppisopimuksen_perusta_koodi_uri     AS oppisopimuksen_perusta_koodi_uri,
+       oh.oppisopimuksen_perusta_koodi_versio  AS oppisopimuksen_perusta_koodi_versio,
+       oh.yksiloiva_tunniste                   AS yksiloiva_tunniste,
+       oh.tyopaikalla_jarjestettava_koulutus_id AS tyopaikalla_jarjestettava_koulutus_id
+FROM osaamisen_hankkimistavat oh
+         JOIN hankittavan_ammat_tutkinnon_osan_osaamisen_hankkimistavat atooh
+              ON oh.id = atooh.osaamisen_hankkimistapa_id
+         JOIN hankittavat_ammat_tutkinnon_osat ato
+              on atooh.hankittava_ammat_tutkinnon_osa_id = ato.id
+UNION
+SELECT oh.id                                   AS id,
```

Comment:
tällasessa voi kai ihan vaan myös laittaa 
```suggestion
SELECT oh.*, concat('pto_', pto.id) AS osa_id, ...
```